d2l-scroll-wrapper.html

```diff
@@ -125,6 +125,7 @@
 
 			attached: function() {
 				this.async(this.checkScrollbar);
+				this.async(this.checkScrollbar);
```

Comment:
This will only be called when the table is attached. So only on initial load or if the table is updated (like while sorting or paging). The method itself shouldn't have too much performance impact. If anything the CSS styling could be updated twice instead of once, and the impact will be minimal since it is only once.